lib/development.conf

```diff
@@ -0,0 +1,192 @@
+                        opencog/attention/libattention-types.so,
+                        opencog/attention/libattention.so,
+                        opencog/embodiment/libembodiment-types.so,
+                        opencog/spacetime/libspacetime-types.so,
+                        opencog/cogserver/modules/python/libPythonModule.so,
+                        opencog/cogserver/modules/libRuleEngineModule.so,
+                        opencog/dynamics/openpsi/libOpenPsi.so
+
+
+# Optional modules, not enabled by default
+#                        opencog/cogserver/modules/benchmark/libbenchmark.so,
+#                        opencog/moldules/events/libatomspacepublishermodule.so
+#                        opencog/attention/libhebbiancreation.so
+#                        opencog/learning/dimensionalembedding/libdimensionalembedding.so
+#                        opencog/viterbi/libviterbi.so
+
+# The below are appropriate if you've done a "sudo make install"
+# The isntall files go into `/usr/local/share/opencog`.
+# If you want to work from the build directory, then uncomment the
+# additional files further down below.
+SCM_PRELOAD           = scm/opencog.scm,
+                        scm/opencog/atom-types.scm,
+                        scm/opencog/nlp.scm,
+                        scm/opencog/nlp/microplanning.scm,
+                        scm/opencog/nlp/sureal.scm,
+                        scm/opencog/nlp/relex2logic.scm,
+                        scm/opencog/nlp/fuzzy.scm,
+                        scm/opencog/nlp/chatbot.scm
+
+#                        scm/config.scm,
+#                        server/scm/config.scm,
+#                        scm/core_types.scm,
+#                        spacetime/spacetime_types.scm,
+#                        nlp/types/nlp_types.scm,
+#                        dynamics/attention/attention_types.scm,
+#                        embodiment/embodiment_types.scm,
+#                        reasoning/pln/pln_types.scm,
+#                        scm/core-docs.scm,
+#                        scm/utilities.scm,
+#                        scm/apply.scm,
+#                        scm/file-utils.scm,
+#                        scm/persistence.scm,
+#                        scm/repl-shell.scm,
+#                        scm/av-tv.scm,
+#                        scm/rule-engine-utils.scm,
+#                        nlp/scm/type-definitions.scm,
+#                        nlp/scm/config.scm,
+#                        nlp/scm/file-utils.scm,
+#                        nlp/scm/nlp-utils.scm,
+#                        nlp/scm/disjunct-list.scm,
+#                        nlp/scm/processing-utils.scm,
+#                        nlp/relex2logic/utilities.scm,
+#                        nlp/relex2logic/rule-helpers.scm,
+#                        nlp/relex2logic/post-processing.scm,
+#                        nlp/lg-dict/utilities.scm,
+#                        nlp/sureal/sureal.scm
+#                        viterbi/viterbi_types.scm
```

Comment:
If you install the scm files all works, but if you are working from the build directory, then it doesn't work. Even if you add opencog directory to load-path.

Tried defining a cmake macro for symlinking the files into the build directory to a similar structure as would be when installed,  so as to add the symlink-directory to load-path on starting cogserver, but wasn't succesful on my first try; `add_custome_command` was a pain. I will try that again after awhile.